modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
+You can use rule-based filtering to control the volume of packets cached in the eBPF flow table. For example, a filter can specify that only packets coming from port 100 should be recorded. Then all packets which do not match the filter are not cached and only the ones that match are cached.
+
+== How ingress and egress traffic filtering works
```

Comment:
```suggestion
== Ingress and egress traffic filtering
```
I tried to eliminate "How" based on the [IBM quick reference guide](https://spaces.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Titles+and+headings).